File: RouteZoekerP/Weg.py
from PIL.ImageFont import truetype
from PIL.ImageDraw import Draw
from PIL.ImageTk   import PhotoImage

class Weg:
    def __init__(self, doel, kosten):
        self.Doel = doel
        self.Kosten = kosten

    def Teken(self, draw, kleur, stad):
        draw.line((stad.Plek, self.Doel.Plek), fill=kleur)

File: RouteZoekerC/Weg.cs
using System.Drawing;

class Weg
{
    public Stad Doel;
    public int Kosten;

    public Weg(Stad doel, int kosten)
    {   Doel = doel;
        Kosten = kosten;
    }
    public void Teken(Graphics gr, Pen pen, Stad stad)
    {   gr.DrawLine(pen, stad.Plek, Doel.Plek);
    }
}